code/game/objects/items/devices/scanners.dm

```diff
@@ -433,7 +433,7 @@ GENE SCANNER
 			render_list += "<span class='notice ml-1'>Subject contains no reagents in their blood.</span>\n"
 		var/obj/item/organ/stomach/belly = M.getorganslot(ORGAN_SLOT_STOMACH)
 		if(belly)
-			if(belly.reagents.reagent_list.len)
+			if(belly.reagents?.reagent_list.len)
```

Comment:
Is there any danger of stomach.reagents being null? are you sure?

Reply:
This was also based on the same assumption, but checking more closely now if stomach exists, reagents definitely should too.